test/e2e/petset.go

```diff
@@ -141,7 +141,7 @@ var _ = framework.KubeDescribe("StatefulSet [Slow] [Feature:PetSet]", func() {
 			ExpectNoError(pst.execInPets(ps, cmd))
 		})
 
-		It("should handle healthy pet restarts during scale [Feature:PetSet]", func() {
+		It("should handle healthy pet restarts during scale", func() {
```

Comment:
References "pet", but I can see other places in this file where that is the case as well. Do we want to change the wording in a follow-up later on?